CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+
+If you're not already familiar with git or Github, here are a couple of friendly tutorials: [First Contributions](https://github.com/firstcontributions/first-contributions), [Open Source Guide](https://opensource.guide/), and [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
+
+## How to file an issue or report a bug
+
+If you see a problem, you can report it in our [issue tracker](https://github.com/aragon/aragon-apps/issues) (or the [Aragon client's](https://github.com/aragon/aragon/issues)).
+
+Please take a quick look to see if the issue doesn't already exist before filing yours.
+
+Do your best to include as many details as needed in order for someone else to fix the problem and resolve the issue.
+
+#### If you find a security vulnerability, do NOT open an issue. Email security@aragon.org instead.
+
+In order to determine whether you are dealing with a security issue, ask yourself these two questions:
+
+- Can I access or steal something that's not mine, or access something I shouldn't have access to?
+- Can I disable something for other people?
+
+If the answer to either of those two questions are "yes", then you're probably dealing with a security issue. Note that even if you answer "no" to both questions, you may still be dealing with a security issue, so if you're unsure, please send a email.
+
+#### If you're interested in the smart contracts underlying Aragon, a [bug bounty program](https://wiki.aragon.org/dev/bug_bounty/) with payouts up to $50,000 is available for rewarding contributors who find security vulnerabilities.
+
+## Fixing issues
+
+1. [Find an issue](https://github.com/aragon/aragon-apps/issues) that you are interested in.
+    - You may want to ask on the issue or on Aragon Chat's [#dev channel](https://aragon.chat/channel/dev) if anyone has already started working on the issue.
+1. Fork and clone a local copy of the repository.
+1. Make the appropriate changes for the issue you are trying to address or the feature that you want to add.
+1. Push the changes to the remote repository.
+1. Submit a pull request in Github, explaining any changes and further questions you may have.
+1. Wait for the pull request to be reviewed.
+1. Make changes to the pull request if the maintainer recommends them.
+1. Celebrate your success after your pull request is merged!
+
+It's OK if your pull request is not perfect (no pull request is).
+The reviewer will be able to help you fix any problems and improve it!
+
+You can also edit a page directly through your browser by clicking the "EDIT" link in the top-right corner of any page and then clicking the pencil icon in the github copy of the page.
+
+## Styleguide and development processes
+
+For the frontends, we use [prettier](https://prettier.io/) and [eslint](https://eslint.org/) to automatically lint and format the project.
```

Comment:
Maybe we could add that we generally have a `npm run lint` task to run ESLint with the right configuration, and that we generally provide a `.eslintrc` and a `.prettierrc` file (not strictly the case everywhere, but it’s easy to fix)?

Reply:
Done in 499d93c.